src/hooks/VideoPlayer.js

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+
+import { VideoContext } from '../shared/VideoContext'
+
+export const VideoPlayer = ({
+  autoplay = false,
+  children,
+  startAt = 0,
+  style,
+}) => {
+  const video = React.useRef()
+  const [isPlaying, setIsPlaying] = React.useState()
+  const pause = () => {
+    setIsPlaying(false)
+    video.current.pause()
+  }
```

Comment:
When do we use newlines between functions and when don't we?

Reply:
Good question. My thoughts here are that each function is a constant and I define the constants at the top of the component then use a newline before the effect, then a newline before the render. Is it tough to read this way?

Reply:
> Is it tough to read this way?

Nope! I think it's just personal preference so let's just ignore this.

But for my two cents, I try to do newlines after `}` that are on their own line that are *not* followed by another `}` so like:
```js
const foo = () => 1
const bar = () => 2 // No curly so no newline
```
```js
const foo = () => {
  return 1
}

const bar = () => { // curly so newline
  return 2
}
```
```js
const foo = () => {
  return () => {
    return 3
  }
} // curly but followed by curly so no newline
```
```js
const foo = () => { return 1 }
const bar = () => { return 2 } // no newline because } isn't on its own line
```

Reply:
ooooo I could get behind this. I know some companies have team-wide style guides. Might be too much to build one of those. Maybe we can make a prettier config for this?

Reply:
That's a great question. I have no idea what prettier offers on this front for config, but yeah maybe we could setup a company wide `eslint`, `tslint`, and/or `prettier.rc` or whatever and have like `gnar-style` but for react/JS